src/rosdep2/platforms/gentoo.py

```diff
@@ -54,10 +58,23 @@ def register_installers(context):
     context.set_installer(PORTAGE_INSTALLER, PortageInstaller())
 
 def register_platforms(context):
+    register_gentoo(context)
+    register_funtoo(context)
+
+def register_gentoo(context):
     context.add_os_installer_key(OS_GENTOO, PORTAGE_INSTALLER)
     context.add_os_installer_key(OS_GENTOO, SOURCE_INSTALLER)
     context.set_default_os_installer_key(OS_GENTOO, PORTAGE_INSTALLER)
 
+def register_funtoo(context):
+    # Funtoo is an alias for Gentoo.
+    (os_name, os_version) = context.get_os_name_and_version()
+    if os_name == OS_FUNTOO and not context.override:
+        print("rosdep detected OS: [%s], aliasing it to [%s]" % (OS_FUNTOO, OS_GENTOO), file=sys.stderr)
+        context.add_os_installer_key(OS_GENTOO, PORTAGE_INSTALLER)
```

Comment:
You probably want to set the OS override like here: https://github.com/ros-infrastructure/rosdep/blob/91816c59a90f22f0044ef0c4843138d8f71d068e/src/rosdep2/platforms/debian.py#L62-L67